src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -292,9 +292,9 @@ def load_command_table(self, _):
         g.custom_command('remove', 'remove_vnet_integration')
 
     with self.command_group('functionapp vnet-integration') as g:
-        g.custom_command('add', 'add_functionapp_vnet_integration', validator=validate_add_vnet, exception_handler=ex_handler_factory())
-        g.custom_command('list', 'list_vnet_integration')
-        g.custom_command('remove', 'remove_vnet_integration')
+        g.custom_command('add', 'add_functionapp_vnet_integration', validator=validate_add_vnet, validator=validate_functionapp_on_containerapp_vnet, exception_handler=ex_handler_factory())
```

Comment:
Are we allowed to have two `validor` params here? I'd think that only one of these values would be used. I would have a call to the `validate_add_vnet` function within the `validate_functionapp_on_containerapp_vnet` function.